dipy/reconst/dti.py

```diff
@@ -1574,7 +1574,8 @@ def _ols_fit_matrix(design_matrix):
 class _NllsHelper:
     r"""Class with member functions to return nlls error and derivative."""
 
-    def err_func(self, tensor, design_matrix, data, weighting=None, sigma=None):
+    @warning_for_keywords()
+    def err_func(self, tensor, design_matrix, data, *, weighting=None, sigma=None):
```

Comment:
probleme with the rebase, to put it back to `def err_func(self, tensor, design_matrix, data, weighting=None, sigma=None):`

Reply:
Done in 092b16850